packages/replicache/src/replicache-impl.ts

```diff
@@ -424,7 +436,7 @@ export class ReplicacheImpl<MD extends MutatorDefs = {}> {
       onClientsDeleted = () => {},
     } = implOptions;
     this.#zero = implOptions.zero;
-    this.auth = auth ?? '';
+    this.#auth = auth ?? '';
```

Comment:
why not undefined? 🤔

Reply:
If I recall correctly it was only so that the type is always string.